Packs/Zimperium/Integrations/ZimperiumV2/ZimperiumV2.yml

```
@@ -0,0 +1,1294 @@
+      description: The id of the rule.
+      type: String
+    - contextPath: Zimperium.PolicyPrivacy.team
+      description: The Team for the policy.
+      type: Unknown
+    - contextPath: Zimperium.PolicyPrivacy.teamId
+      description: The Team ID the Policy associated with.
+      type: Unknown
+  - arguments:
+    - description: The identifier of the policy. Can be retrieved using zimperium-policy-group-list in the Zimperium.PolicyGroup.trmId field.
+      name: policy_id
+      required: true
+    description: Get threat policy by its identifier.
+    name: zimperium-policy-threat-get
+    outputs:
+    - contextPath: Zimperium.PolicyThreat.id
+      description: The identifier of the policy.
+      type: String
+    - contextPath: Zimperium.PolicyThreat.accountId
+      description: The account identifier of the policy.
+      type: String
+    - contextPath: Zimperium.PolicyThreat.androidJsonHash
+      description: The Android JSON hash.
+      type: String
+    - contextPath: Zimperium.PolicyThreat.androidProtoHash
+      description: The Android Proto Hash.
+      type: String
+    - contextPath: Zimperium.PolicyThreat.assigned
+      description: Whether the policy is assigned.
+      type: Boolean
+    - contextPath: Zimperium.PolicyThreat.created
+      description: The date and time the policy threat was created.
+      type: Date
+    - contextPath: Zimperium.PolicyThreat.deploymentDate
+      description: The date when the policy deployment occurred.
+      type: Date
+    - contextPath: Zimperium.PolicyThreat.global
+      description: Whether the policy setting are global.
+      type: Boolean
+    - contextPath: Zimperium.PolicyThreat.groups
+      description: The groups the policy associated with.
+      type: Unknown
+    - contextPath: Zimperium.PolicyThreat.iosJsonHash
+      description: IOS Json Hash.
+      type: String
+    - contextPath: Zimperium.PolicyThreat.iosProtoHash
+      description: IOS Proto Hash.
+      type: String
+    - contextPath: Zimperium.PolicyThreat.isDeployed
+      description: Whether the PolicyThreat is deployed or not.
+      type: Boolean
+    - contextPath: Zimperium.PolicyThreat.modified
+      description: The date and time when the policy was modified.
+      type: Date
+    - contextPath: Zimperium.PolicyThreat.name
+      description: The name of the policy.
+      type: String
+    - contextPath: Zimperium.PolicyThreat.rules
+      description: The policy rules list.
+      type: Unknown
+    - contextPath: Zimperium.PolicyThreat.rules.id
+      description: The ID of the Policy rule.
+      type: String
+  - arguments:
+    - description: The identifier of the policy. Can be retrieved using zimperium-policy-group-list in the Zimperium.PolicyGroup.phishingPolicyId field.
+      name: policy_id
+      required: true
+    description: Get threat policy by its identifier.
+    name: zimperium-policy-phishing-get
+    outputs:
+    - contextPath: Zimperium.PolicyPhising.id
```

should be:
```suggestion
    - contextPath: Zimperium.PolicyPhishing.id
```
don't forget to update in the code as well